tests/test_pages.py

```diff
@@ -0,0 +1,98 @@
+# pylint: disable=redefined-outer-name
+from __future__ import print_function
+
+from collections import namedtuple
+
+import pytest
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
```

Comment:
I find the `EC` name irritating (it kinda pretends to be a static var?), why renaming it at all?

Reply:
Cowardly copied from https://selenium-python.readthedocs.io/waits.html#explicit-waits

Reply:
heh, fine :)

Reply:
I tend to agree. It pretends to be a static variable and requires me to do an extra step of translating the code when reading to know what EC stands for and what it does.